operations/model.go

```diff
@@ -91,6 +91,23 @@ func NewClientSettings(fn string) (*ClientSettings, error) {
 	}
 	conf.LoadedFrom = path
 
+	// check for a local settings file
+	localPath, err := findConfigFilePath(".evergreen.yml")
```

Comment:
Evergreen already looks for a file in a project's directory, often (in 52 projects) called ".evergreen.yml".

Reply:
I wondered about that I couldn't remember the default "project config" location (since I can't create projects anymore :D)

Do you have a suggested filename that would work better? Something like `.evergreen.local.yml`?

Reply:
Yeah, I think that works. Could you make that a string constant?

Reply:
Done.